README.md

```diff
@@ -43,9 +43,11 @@ For more information on [how to get involved on the CHAOSS website](https://chao
 
 ## Collecting Data
 
-Augur supports ```Python3.7``` through ```Python3.11``` on all platforms. ```Python3.12``` and above do not yet work because of machine learning worker dependencies. On OSX, you can create a ```Python3.11``` environment, by running:
+Augur officially supports ```Python3.11``` on all platforms. ```Python3.12``` and above do not yet work because of machine learning worker dependencies. We recommend using [uv](https://github.com/astral-sh/uv) to manage your Python environment:
 ```
-$ python3.11 -m venv path/to/venv
+$ uv venv --python 3.11
+$ source .venv/bin/activate  # On Unix/macOS
+$ .venv\Scripts\activate     # On Windows
```

Comment:
this isnt necessary with uv, it handles creating and activating an env for you

im not sure what should go here. `uv run augur` would make some sense, but i have yet to test if that actually works. the Docker quick start in the readthedocs is the recommended way to set augur up currently

Reply:
Can you open a new issue for the issues you describe with bringing the migrations up on main?

The scope of this PR is just updating docs essentially. IMO i think the docker based workflow by default makes the most sense for most people, at least until there are updated scripts or other similarly easy ways newcomers can do a manual setup.

Reply:
I agree that the docker based setup should be primary now.